indexer-app/src/cmr/indexer/data/concepts/collection/keyword.clj

```diff
@@ -3,11 +3,25 @@
   (:require
    [clojure.string :as str]
```

Comment:
This should be changed from str to string.  I noticed that you are using str below in your code.